src/stateManagement/toolState.js

```diff
@@ -30,18 +30,18 @@ function getElementToolStateManager(element) {
  * @method addToolState
  *
  * @param  {HTMLElement} element  The element.
- * @param  {string} toolType      The toolType of the state.
+ * @param  {string} toolName      The toolName of the state.
  * @param  {Object} measurementData The data to store in the state.
  * @returns {undefined}
  */
-function addToolState(element, toolType, measurementData) {
+function addToolState(element, toolName, measurementData) {
   const toolStateManager = getElementToolStateManager(element);
 
-  toolStateManager.add(element, toolType, measurementData);
+  toolStateManager.add(element, toolName, measurementData);
 
   const eventType = EVENTS.MEASUREMENT_ADDED;
   const eventData = {
-    toolType,
+    toolName,
```

Comment:
This is a breaking api change. I'm happy to change this as its technically correct, although perhaps we should have a deprecated alias with `toolType` Either that or we have its v4 only. What do you think @dannyrb ?

Reply:
I think v4 is coming down the pipe soon enough that we should just do it there, and make a note of it in our breaking changes / migration guide.